### ISSUE TITLE
Clean way to add ...-allow-origin to non-preflight requests

### ISSUE THREAD

**Issue by theashguy** [lawliet89/rocket_cors#38]
Hey there guys, 

I may have this one completely wrong but by the looks of it the fairing only handles automagically generating the preflight / OPTIONS request and not the correct headers for other request types. Its not obvious to me the idiomatic way to achieve this. I could obviously go through request but I'm adding CORS to an already existing application so this is going to be a long winded task. 

Basically wanted to check if there was some kind of exploiting default headers in rocket rather than manually tweaking all the existing code.

Cheers.
Ash.

**Comment by lawliet89** [comment]
There is unfortunately no automagical way to do so. It can be implemented.

May I know your use case? I don't really see why you might want to add those headers to non CORS requets.

**Comment by lawliet89** [comment]
Closing due to lack of response. Please reopen otherwise.
